docs/integrations/pydantic.md

```diff
@@ -265,6 +265,41 @@ input_data = UserInput(id='abc', name='Jake')
 instance = input_data.to_pydantic()
 ```
 
+### Constrained types
+
+Strawberry supports [pydantic constrained types](https://pydantic-docs.helpmanual.io/usage/types/#constrained-types).
+Note that constraint is not enforced in the graphql type. Thus, we recommend always working on the pydantic
+type such that the validation is enforced.
+
+```python+schema
+from pydantic import BaseModel, conlist
+import strawberry
+
+class Example(BaseModel):
+    friends: conlist(str, min_items=1)
+
+@strawberry.experimental.pydantic.input(model=Example, all_fields=True)
+class ExampleGQL:
+    ...
+
+@strawberry.type
+class Query:
+    @strawberry.field()
+    def test(self, example: ExampleGQL) -> None:
+        # friends may be an empty list here
+        print(example.friends)
+        # calling to_pydantic() runs the validation and raises
+        # an error if friends is empty
+        print(example.to_pydantic().friends)
+
+schema = strawberry.Schema(query=Query)
+
+---
+input ExampleGQL {
+  friends: [String!]!
+}
```

Comment:
can we also add the schema for the query? 😊

Reply:
done!